creme/tree/hoeffding_tree_classifier.py

```diff
@@ -0,0 +1,340 @@
+from ._split_criterion import HellingerDistanceCriterion
+from ._nodes import SplitNode
+from ._nodes import LearningNode
+from ._nodes import ActiveLeaf
+from ._nodes import ActiveLearningNodeMC
+from ._nodes import ActiveLearningNodeNB
+from ._nodes import ActiveLearningNodeNBA
+from ._nodes import InactiveLearningNodeMC
+
+
+class HoeffdingTreeClassifier(DecisionTree, base.Classifier):
+    """ Hoeffding Tree or Very Fast Decision Tree classifier.
+
+    Parameters
+    ----------
+    grace_period
+        Number of instances a leaf should observe between split attempts.
+    split_criterion
+        | Split criterion to use.
+        | 'gini' - Gini
+        | 'info_gain' - Information Gain
+        | 'hellinger' - Helinger Distance
+    split_confidence
+        Allowed error in split decision, a value closer to 0 takes longer to decide.
+    tie_threshold
+        Threshold below which a split will be forced to break ties.
+    leaf_prediction
+        | Prediction mechanism used at leafs.
+        | 'mc' - Majority Class
+        | 'nb' - Naive Bayes
+        | 'nba' - Naive Bayes Adaptive
+    nb_threshold
+        Number of instances a leaf should observe before allowing Naive Bayes.
+    nominal_attributes
+        List of Nominal attributes identifiers. If empty, then assume that all numeric attributes
+        should be treated as continuous.
+    **kwargs
+        Other parameters passed to river.tree.DecisionTree.
+
+    Notes
+    -----
+    A Hoeffding Tree [^1] is an incremental, anytime decision tree induction algorithm that is
+    capable of learning from massive data streams, assuming that the distribution generating
+    examples does not change over time. Hoeffding trees exploit the fact that a small sample can
+    often be enough to choose an optimal splitting attribute. This idea is supported mathematically
+    by the Hoeffding bound, which quantifies the number of observations (in our case, examples)
+    needed to estimate some statistics within a prescribed precision (in our case, the goodness of
+    an attribute).
+
+    A theoretically appealing feature of Hoeffding Trees not shared by other incremental decision
+    tree learners is that it has sound guarantees of performance. Using the Hoeffding bound one
+    can show that its output is asymptotically nearly identical to that of a non-incremental
+    learner using infinitely many examples.
+
+    Implementation based on MOA [^2].
+
+    References:
+    .. [1] G. Hulten, L. Spencer, and P. Domingos. Mining time-changing data streams.
+       In KDD’01, pages 97–106, San Francisco, CA, 2001. ACM Press.
+
+    .. [2] Albert Bifet, Geoff Holmes, Richard Kirkby, Bernhard Pfahringer.
+       MOA: Massive Online Analysis; Journal of Machine Learning Research 11: 1601-1604, 2010.
+
+    Examples
+    --------
+    >>> # Imports
+    >>> from skmultiflow.data import SEAGenerator
+    >>> from skmultiflow.trees import HoeffdingTreeClassifier
+    >>>
```

Comment:
You don't need empty `>>>`s, just remove them :)

Reply:
Although the examples are WIP, I removed the empty `>>>` as a reminder.